.travis.yml

```diff
@@ -8,4 +8,6 @@ branches:
     - master
 
 script:
-- ./test.bash
+- curl https://raw.githubusercontent.com/GoogleCloudPlatform/runtimes-common/master/structure_tests/ext_run.sh > ext_run.sh
```

Comment:
Can you pin this to a specific commit?  I like reproducible builds/tests, although I realize that every other part of this build can slide versions around.